airflow/providers/google/suite/hooks/drive.py

```diff
@@ -243,8 +292,21 @@ def upload_file(
             .create(body=file_metadata, media_body=media, fields="id", supportsAllDrives=True)
             .execute(num_retries=self.num_retries)
         )
-        self.log.info("File %s uploaded to gdrive://%s.", local_location, remote_location)
-        return file.get("id")
+        file_id = file.get("id")
+
+        upload_location = remote_location
+
+        if folder_id != "root":
+            try:
+                upload_location = self._resolve_file_path(folder_id)
+            except GoogleApiClientError as e:
+                self.log.warning("A problem has been encountered when trying to resolve file path: ", e)
+
+        if show_full_target_path:
+            self.log.info("File %s uploaded to gdrive://%s.", local_location, upload_location)
+        else:
+            self.log.info("File %s has been uploaded successfully to gdrive", local_location)
```

Comment:
i think this parameter is redundant?
It doesn't give any additional value. Lets just print the full path. if someone doesn't need the full path they can just remove `gdrive://` from the path

Reply:
@eladkal -> this is what I asked for as an escape hatch. The problem is that there might be potentially problems that might lead to  exception raised when "_resolve_file_path". is called (I thought for example if you have no permission to read the whole path. GDrive is not a "normal" filesystem and traversing up the directory might in some cases turn problematic. Since we ONLY need that to log the path, this is something that we might want to disable.

Reply:
👌